pipelines/regobs-web-build-release.yml

```diff
@@ -64,6 +67,32 @@ stages:
               command: custom
               customCommand: 'run create-version-file'
 
+          - task: oneLuckiDevJson2Variable@1
+            displayName: 'Create environment variables from version.json file'
+            inputs:
+              jsonFile: '$(Build.SourcesDirectory)/src/environments/version.json'
+              shouldPrefixVariables: true
+
+          - task: CopyFiles@2
+            displayName: 'Ta med sentry.properties i bygget. Trengs for å lage release i Sentry'
```

Comment:
ser bra ut og det fungerer som er viktigst. jeg bare lurte på om vi burde være konsekvente å bruke samme språk for å kalle tasker. ser ut at vi bruker engelsk på alt annet så kanskje ok å ha engelsk navn her og?

Reply:
Jeg vil heller ha norsk på alt;-)